### ISSUE TITLE
PDF smaller then SVG

### ISSUE THREAD

**Issue by davidgatti** [marcbachmann/node-html-pdf#86]
![screen shot 2015-11-17 at 1 51 58 pm](https://cloud.githubusercontent.com/assets/411114/11220303/98431842-8d5f-11e5-8bb3-292b81a5a361.png)

I have this SVG file created in illustrator that i use to convert in to a PDF. The end result is bellow: 

![screen shot 2015-11-17 at 1 51 55 pm](https://cloud.githubusercontent.com/assets/411114/11220280/77326a54-8d5f-11e5-99ac-13780af63673.png)

Any idea why the elements are smaller? I converted the units from points to pixels but that didn't fix the problem.

**Comment by marcbachmann** [comment]
It looks like only the table has a wrong width. Did you set the width of it to 100%?

**Comment by davidgatti** [comment]
@marcbachmann this is a SVG file there is no such concept as 100% in illustrator, well at least as far as i know :) 

Plus it is not only the table, if you look closely, also the position of the text have a different position. Basically everything is more narrow.

About the DPI, it is only used in print when you want to export a vector image for example in to a raster image and you want the right proportion but with resolution that will give a good quality print. 

I'm attaching the top fragment of the SVG file. Maybe there is something wrong/unnecessary that Illustrator adds to the file?

```xml
<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 width="595.2px" height="841.7px" viewBox="0 0.2 595.2 841.7" style="enable-background:new 0 0.2 595.2 841.7;"
	 xml:space="preserve">
<text transform="matrix(1 0 0 1 197.79 51.1347)" class="st0 st1">Invocie Nr. </text>
<text id="invoiceNr" transform="matrix(1 0 0 1 331.9414 51.1347)" class="st0 st1">#####</text>
<text transform="matrix(1 0 0 1 36.9839 97.455)" class="st0 st2">From:</text>
<text transform="matrix(1 0 0 1 20.9624 110.7441)" class="st0 st2">Address:</text>
```

**Comment by marcbachmann** [comment]
I assume that vectors are skewed but text is not (because it's not outlined).
Can you make create a square in the svg to test whether it's displayed as square.
Do you pass the svg directly to the render function or do you embed it in the html?

I mentioned ppi (pixel per inch, not dots per inch) to make sure you're using the right conversion in a case you're using the metric system to define the width, but apparently you are using pixels, so you can ignore that.

**Comment by davidgatti** [comment]
OK, will some experiments. by the way @marcbachmann, one question before i start. I just discovered that in the US you don't use A4 format, but rather the Letter format which is slightly different. Could this be the issue? 

The SVG that i have have the size of the A4 paper

**Comment by marcbachmann** [comment]
Sure this can be an issue. Currently the default is A4 in this library. So
that's the right one for your use case. But check the size of your svg ;)

Greetings from Zurich

**Comment by davidgatti** [comment]
Zurich you say ;) 

ok, then ;)

**Comment by davidgatti** [comment]
OK @marcbachmann, I did some test and now my PDF is better but not as it should be. 

**Things to know**

- I convert a clean SVG file. Meaning is not wrapped in html and body tag (adding them dose nothing)
- Settings options, for example: the resolution but type (a4 etc) or with etc dose nothing 

**What almost worked**

I set the size in inches in the SVG tag

```
<svg version="1.1"
id="Layer_1" 
xmlns="http://www.w3.org/2000/svg" 
xmlns:xlink="http://www.w3.org/1999/xlink" 
x="0px" y="0px"
width="8.5in" height="11in" 
viewBox="0 0 612 792" 
xml:space="preserve">
```

**The result**

![screen shot 2015-11-23 at 10 42 26 pm](https://cloud.githubusercontent.com/assets/411114/11350927/3d64d864-9234-11e5-88ae-0055933bae22.png)

The first image shows what your module created and as you can see the inches are off. The second image shows a PDF created by Illustrator and has the exact resolution.

![screen shot 2015-11-23 at 10 42 30 pm](https://cloud.githubusercontent.com/assets/411114/11350965/57d8444c-9234-11e5-844d-896f74a729d7.png)

**Question**

Do you have any idea how to make and exact size?

**Comment by davidgatti** [comment]
Another thing, i set the background to red to see what is happening. and basically i realized that the resolution in inches that i wrote in the SVG tag has no effect on the size of the PDF but on the "iframe" size of the embedded SVG. 

Plus as you can see there are white margin on the left and bottom. 

![screen shot 2015-11-23 at 11 04 16 pm](https://cloud.githubusercontent.com/assets/411114/11351382/a7a3d048-9236-11e5-87c9-457e968256f7.png)

**Comment by davidgatti** [comment]
@marcbachmann any news on this issue? :)

**Comment by davidgatti** [comment]
OK, I managed to fix the problem at 99.999% :) I did embed the SVG in the following HTML 

```html
<html>
<head>
    <meta charset="utf-8">
    <style>
    body {
        margin: 0;
    }
    </style>
</head>
<body>SVG</body>
</html>
```

The end result is have a tiny margin at the bottom with the height `12.031in` fine tuning the hight at this resolution gives a 2nd page. 

```sag
<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 width="8.5in" height="12.031in" viewBox="0 0 612 792" xml:space="preserve" style="background-color: red">
```

@marcbachmann if in the future you could make the module work without embedding the SVG in to HTML and keeping the margins at `0`, it would be perfect :) But I still thank you for the grate module :)
